pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1973,7 +1977,7 @@ func rktRun(domainName string, ContainerImageID string, xenCfgFilename string) (
 		"--dir=" + types.PersistRktDataDir,
 		"--insecure-options=image",
 		"run",
-		ContainerImageID,
+		"/persist/downloads/appImg.obj/" + ContainerImageID + ".aci",
```

Comment:
why is not a constant? I mean "/persist/downloads/appImg.obj/" ?

Reply:
@rvs I suspect the cleanup/delete code needs to be able to find the images, and for containers we don't have a hash to name them hence we use the image UUID.